src/java/org/apache/cassandra/db/streaming/ComponentManifest.java

```diff
@@ -27,15 +27,11 @@
 import org.apache.cassandra.io.sstable.Descriptor;
 import org.apache.cassandra.io.util.DataInputPlus;
 import org.apache.cassandra.io.util.DataOutputPlus;
+import org.apache.cassandra.io.util.File;
 
 import java.io.IOException;
 import java.util.*;
```

Comment:
nit: heh, these were out of order before your patch, but might as well fix that here :)